zerver/webhooks/sentry/doc.md

```diff
@@ -1,38 +1,36 @@
+# Zulip Sentry integration
+
 Get Zulip notifications for the issues in your Sentry projects!
 
 This integration supports Sentry's Node, Python, and Go
 [platforms](https://sentry.io/platforms/).  [Contact
 us](/help/contact-support) if a platform you care about is missing.
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
+
+    !!! tip ""
 
-    The default topic, if not set in the URL, will be the title of the
-    issue or event.
+        The default topic, if not set in the URL, will be the title of the
```

Comment:
I think we can rewrite this to remove the "if not set in the URL" as that's what default means?